content/docs/release-notes/22-10-0/_index.md

```diff
@@ -0,0 +1,264 @@
+
+**Physics**
+- Setting the Cylinder Collider component's radius or height to zero causes the O3DE Editor to crash. This is a known issue that will be fixed in development. Until it's resolved, avoid setting the radius or height to zero. [#12147](https://github.com/o3de/o3de/issues/12147)
+- The physics material library asset has been removed now supports individual physics material assets. If you work with AutomatedTesting, atom-sampleviewer, multiplatyersample or netsoaktest, you don't need to do anything -- all physics material assets are already converted.
+
+  For any other project, one person must convert the assets by following these steps:
+
+  1. Run Editor. Do not open any level, just close the "Welcome to O3DE" screen.
+  1. Wait until Asset Processor has finished processing all the assets.
+  1. Open Console by clicking **Tools** > **Console**.
+  1. Enter the following console command to convert old physics material libraries: `ed_physxConvertMaterialLibrariesIntoIndividualMaterials`
+  1. Wait until Asset Processor has finished processing all the assets.
+  1. Enter the following console command to fix prefabs and fbx manifests that reference old assets: `ed_physxFixAssetsUsingPhysicsLegacyMaterials`
+  1. Wait until Asset Processor has finished processing all the assets.
+  1. Commit the source asset changes into Github.
+
+**Viewport**
+- The Component Mode Switcher may suffer from instability on Linux. If this is experienced you can disable the Switcher in the Settings Registry by adding the following: 
+
+  ```
+  {
+      "Amazon": {
+          "Preferences": {
+              "Editor": {
+                  "ComponentSwitcherEnabled": false
+              }
+          }
+      }
+  }
+  ```
+
+**Animation**
+
+- Do not include `.` in names for animations in you DCC tool as this can fail at import time. [#12342](https://github.com/o3de/o3de/issues/12342)
+
+- Some animations crash in the scene settings tool when a level isn't opened. Ensure a level is opened before accessing the scene settings tool. [#12414](https://github.com/o3de/o3de/issues/12414)
+
+**UI**
+
+- In the UI Editor, the Curve Editor is not functional. [#11616](https://github.com/o3de/o3de/issues/11616)
+
+- In the UI Editor, modifying keys may crash the Editor. [#12219](https://github.com/o3de/o3de/issues/12219)
+
+**Prefab**
+
+- After creating and deleting prefabs, you must save your level twice.
+
+- After prefab operations, you may encounter warnings that say "Some of the patches were not successfully applied". 
+  - For prefab deletion, it has been fixed in development. [#11756](https://github.com/o3de/o3de/issues/11756)
+  - For prefab creation, it will be fixed in development. [#12013](https://github.com/o3de/o3de/issues/12013)
+
+**Editor**
+
+- The O3DE Editor splash screen displays "Development Build" instead of the version number. [#9519](https://github.com/o3de/o3de/issues/9519)
```

Comment:
This is unique to Linux, for this specific situation
```suggestion
- On linux, the O3DE Editor splash screen displays "Development Build" instead of the version number if development and release are installed together. [#9519](https://github.com/o3de/o3de/issues/9519)
```